tests/test_enochecker.py

```diff
@@ -290,16 +224,6 @@ def test_useragents():
     assert first_agent != checker.http_useragent
 
 
-@temp_storage_dir
-def test_exceptionHandling(capsys):
```

Comment:
No longer needed?

Reply:
I am not sure what this was doing in the first place, as far as I could tell there were no assertions so I just got rid of it